src/main/java/org/jitsi/videobridge/rest/HandlerImpl.java

```diff
@@ -1023,10 +1017,6 @@ else if (target.startsWith(STATISTICS))
             statisticsRequestHandler.handleStatsRequest(
                     target, request, response);
         }
-        else if (target.startsWith(DEBUG))
```

Comment:
With this code gone and DebugRequestHandler gone, is HandlerImpl.DEBUG now no longer needed?

Reply:
yup, fixed